packages/commons/src/auth/jwk.ts

```diff
@@ -64,3 +66,17 @@ export function sortJWK(jwk: JsonWebKey): JsonWebKey {
       {}
     );
 }
+
+export function getJwksClient(): JwksClient[] {
+  const config = JWTConfig.parse(process.env);
+  return config.wellKnownUrls.map((url) =>
+    jwksClient({
+      cache: true,
+      cacheMaxEntries: 50,
+      timeout: 30000,
+      cacheMaxAge: 3600000, // 60 minutes
```

Comment:
@beetlecrunch was suggesting in the thread to set the cache max age to 1 minute (`60000`) in the thread where we discussed this

Reply:
In agreement with @galales let's keep the default value 10 minutes.